test/FixedDataTableRoot-test.js

```diff
@@ -12,7 +12,7 @@ import {
   isElement,
 } from 'react-dom/test-utils';
 
-import { Table, Column } from '../src/FixedDataTableRoot';
+import { Table, Column } from '../src/index';
```

Comment:
`FixedDataTableRoot.js` which was sort of serving as a barrel file is now renamed to `index.js`